### PR TITLE
Batch & Session APIs

### PR DESCRIPTION
**Pull request by rathorevaibhav**
### Summary
More API endpoints for:
1. Batch bulk update sessions
2. Session bulk update batches
3. Correction in batch session migration for batch instead of the user. Will have to refresh migrations and recreate DB.
4. Hyphenated URLs

### CODE REVIEW

File: lib/dbservice/sessions.ex [hunk 1]
```diff
@@ -102,6 +102,23 @@ defmodule Dbservice.Sessions do
     Session.changeset(session, attrs)
   end
 
+  @doc """
+  Updates the users mapped to a batch.
+  """
+  def update_batches(session_id, batch_ids) when is_list(batch_ids) do
```
Comment (pritamps) [inline lib/dbservice/sessions.ex:108]: There's no user anywhere in this method, though users are mentioned in the doc. This is intentional?

File: lib/dbservice/batches.ex [hunk 2]
```diff
@@ -118,4 +118,21 @@ defmodule Dbservice.Batches do
     |> Batch.changeset_update_users(users)
     |> Repo.update()
   end
+
+  @doc """
+  Updates the sessions mapped to a batch.
+  """
+  def update_sessions(batch_id, session_ids) when is_list(session_ids) do
```
Comment (pritamps) [inline lib/dbservice/batches.ex:125]: The Swagger PR has API docs, no? I feel like it will be needed :D

File: lib/dbservice/sessions.ex (hunk 1 above)
Comment (rathorevaibhav) [reply lib/dbservice/sessions.ex:108]: Sorry copy/paste error. I'll update.

File: lib/dbservice/batches.ex (hunk 2 above)
Comment (rathorevaibhav) [reply lib/dbservice/batches.ex:125]: Yes, coming after a couple of PRs.

**Review by pritamps — APPROVED** [review]

**Pull request merged**